### PR TITLE
PR: Landing page

### PR DESCRIPTION
**Pull request by Jayesh3636**
- Added Landing with main component tab

https://user-images.githubusercontent.com/91378157/153184782-740dfa67-1a20-4c08-973c-2adcf5cda99f.mp4

### CODE REVIEW

File: Components/library.css [hunk 1]
```diff
@@ -0,0 +1,190 @@
+/* colors */
+@import url('https://fonts.googleapis.com/css2?family=Karla:wght@200;300;400;500;600;700;800&family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+/* <link rel="stylesheet" href="./library.css">
```
Comment (pnchinmay) [inline Components/library.css:3]: Remove these comments

**Review by pnchinmay — APPROVED** [review]

**Review by pnchinmay — APPROVED** [review]
UI looks great :100:

File: Components/library.css [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    <link rel="stylesheet" href="../reset.css">
+    <title>DarkBucket</title>
+    <link rel = "icon" href = "images/ICON.png" type = "image/x-icon"> */
```
Comment (pnchinmay) [inline Components/library.css]: Remove these comments

**Review by pnchinmay — CHANGES_REQUESTED** [review]

**Review by pavan6115 — APPROVED** [review]
Awesome UI

**Pull request merged**